### ISSUE TITLE
feat: improve accessibility to meet WCAG standards

### ISSUE THREAD

**Issue by techfg** [ues-io/uesio#4432]
Currently, ues.io (website, studio, runtime, etc.) do not meet WCAG standards in many situations.  For example:

1. Contrast ratios
2. Image alternate text
3. Form element labels
4. Keyboard access for scrollable regions
5. Links must have discernible text (e.g., icon buttons)
6. Nested interactive controls
7. Touch targets at least 24px
8. Landmarks 

A decision needs to be made on which WCAG standard should be met (e.g., 2.1, 2.2, etc.) and then updates made to achieve.

Reference:
1. https://www.w3.org/WAI/standards-guidelines/wcag/
2. https://www.etsi.org/human-factors-accessibility/en-301-549-v3-the-harmonized-european-standard-for-ict-accessibility
[latest version](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf)
